### PR TITLE
cloud install > do this after cloud-databases, cloud-tables, etc

### PR DESCRIPTION
**Pull request by ljsinclair**
updated cloud intro
new cloud setup
new cloud login
new include

### CODE REVIEW

File: cloud/_posts/2022-11-09-cloud-introduction.md [hunk 1]
```diff
@@ -2,38 +2,35 @@
...
-- Data ingest that allows you to push records when ready and without having to open up your datacenter
+You can test FeatureBase cloud free for 21 days. After this time your account will be limited to login only.
 
-## Frequently asked questions
+| Feature | Limitations |
+|---|---|
+| Create account | Create the account owner and user accounts. |
+| Create databases and tables | One database per trial account |
+| Import (ingest) data from data source | One data source per trial account |
+| Run queries on data | Queries are limited to 30 seconds of execution time and/or 6MB of data during the trial. |
```
Comment (gthrone25) [inline cloud/_posts/2022-11-09-cloud-introduction.md:24]: We can get rid of this line now!!!

File: cloud/_posts/2022-11-14-fbc-part1-signup.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+Sign up to FeatureBase cloud using these instructions.
+
+## Before you begin
+
+* [Learn more about FeatureBase cloud](/cloud/cloud-introduction)
+* [Learn about the 21 day trial period](/cloud/cloud-introduction#cloud-trial)
+* [Read the terms of service](https://www.featurebase.com/cloud-terms)
+* Have contact details and a password ready
```
Comment (gthrone25) [inline cloud/_posts/2022-11-14-fbc-part1-signup.md:12]: I think we can cut this line out, unless you think it's necessary.

File: cloud/_posts/2022-11-14-fbc-part1-signup.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Signing up for Featurebase cloud
+
+This is a 21 day trial period
+
+1. Click [Sign up for cloud trial](https://cloud.featurebase.com/signup)
+2. Check your email for the confirmation message to complete signup.
+
+## Next step
+
+* [Login to FeatureBase Cloud](https://cloud.featurebase.com/login)
```
Comment (gthrone25) [inline cloud/_posts/2022-11-14-fbc-part1-signup.md:23]: Why do we link the actual login vs the page you built? I do prefer this and getting rid of the other page, but I'm guessing you have some thinking around this?

**Review by gthrone25 — COMMENTED** [review]
Looks good. I think the ordering of pages is off though. It should be introduction -> signup -> login (if login still exists :) )

**Comment by gthrone25** [comment]
@andreacappelletti97 I’m tagging you to review the following doc changes. Please leave a review that comments on the following:
 
- [ ] Build the branch locally (see README for instructions)
- [ ] Content of new pages is accurate and clear (i.e. things are described correctly and are understandable)
- [ ] Content of new pages contains all information from removed pages (if applicable)
- [ ] Content of pages is comprehensive (i.e. all relevant content and features are documented and nothing is missing)
- [ ] Check for broken links in the page (Note some links may be intentionally broken in cases where many PRs are planned to go in simultaneously)
- [ ] Cloud Only - Identify if any links in the product are broken and need to be changed
- [ ] Any other errors found (grammatical, spelling, consistent tense, etc.)

File: cloud/_posts/2022-11-09-cloud-introduction.md [hunk 4]
```diff
@@ -2,38 +2,35 @@
...
 
-- Data ingest that allows you to push records when ready and without having to open up your datacenter
+You can test FeatureBase cloud free for 21 days. After this time your account will be limited to login only.
 
-## Frequently asked questions
+| Feature | Limitations |
+|---|---|
+| Create account | Create the account owner and user accounts. |
+| Create databases and tables | One database per trial account |
+| Import (ingest) data from data source | One data source per trial account |
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/_posts/2022-11-09-cloud-introduction.md:23]: Yeah, this is something I wanted to talk with you (and everyone else) about.

I raised a question on Stackoverflow to discuss:
https://stackoverflowteams.com/c/molecula/questions/220

File: cloud/_posts/2022-11-09-cloud-introduction.md (hunk 1 above)
Comment (ljsinclair) [reply cloud/_posts/2022-11-09-cloud-introduction.md:24]: I've simplified this with an include used elsewhere in the product which I think makes this simpler

File: cloud/_posts/2022-11-14-fbc-part1-signup.md (hunk 2 above)
Comment (ljsinclair) [reply cloud/_posts/2022-11-14-fbc-part1-signup.md:12]: I'm doing this in PR https://github.com/molecula/documentation/pull/268

File: cloud/_posts/2022-11-14-fbc-part1-signup.md (hunk 3 above)
Comment (ljsinclair) [reply cloud/_posts/2022-11-14-fbc-part1-signup.md]: Sound thinking? No. I missed it in the huge sets of changes over the past few weeks and it's another good reason to break things down smaller :D.

Comment (ljsinclair) [reply cloud/_posts/2022-11-14-fbc-part1-signup.md]: Fixed (plus others) https://github.com/molecula/documentation/pull/268/commits/c4391fe6101d72c62b2c46cf908139626bef0ba7

File: cloud/_posts/2022-11-09-cloud-introduction.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gthrone25) [reply cloud/_posts/2022-11-09-cloud-introduction.md]: ok we discuss there but don't have to let it stop this from going in

**Comment by andreacappelletti97** [comment]
> 

Hi @gthrone25, here you can find our review
cc: @rutherford-jasper 

*  Build the branch locally (see README for instructions)
We had to install a specific version of bundler, maybe it could be great to specify it `sudo gem install bundler:2.3.9`

* Content of new pages is accurate and clear (i.e. things are described correctly and are understandable)
    * Cloud Introduction
        * Cloud Trial
            * The feature/limitation table is not that much clear, we were asking ourselves what are we missing? Maybe a comparison with the full package (not trial) would be more helpful
            * The table style could be much clearer, only lines separation between different rows can be a little confusing 
    * How do I signUp on FeatureBase Cloud?
        * Everything’s clear here
    * Login to FeatureBase Cloud
        * Everything’s clear here

* Content of new pages contains all information from removed pages (if applicable)
    * Cloud Introduction
        * FeatureBase overview section is not covered anymore (we think that this was intentional)

* Content of pages is comprehensive (i.e. all relevant content and features are documented and nothing is missing)
    * Everything looks good under this point 
* Check for broken links in the page (Note some links may be intentionally broken in cases where many PRs are planned to go in simultaneously)
    * Cloud Introduction
        * Cloud Trial
            * Contact FeatureBase support: link is broken
    * How do I signUp on FeatureBase Cloud?
        * Before you begin
            * Sign-up to FeatureBase Cloud: link is broken

* Cloud Only - Identify if any links in the product are broken and need to be changed
    * See the section above, nothing more to report

* Any other errors found (grammatical, spelling, consistent tense, etc.)
    * Checked with Grammarly and everything looks correct

Additional notes
* Typo Minmia should be Minima in the readme of the repository Theme = Minmia (Jekyll default)

File: cloud/_posts/2022-11-30-cloud-introduction.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## Cloud trial
+
+You can test FeatureBase cloud free for 21 days. After this time your account will be limited to login only.
+
+| Feature | Limitations |
+|---|---|
+| Create account | Create the account owner and user accounts. |
+| Create databases and tables | One database per trial account |
+| Import (ingest) data from data source | One data source per trial account |
+| Run queries on data | Queries are limited to 30 seconds of execution time and/or 6MB of data during the trial. |
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/_posts/2022-11-30-cloud-introduction.md:24]: this content removed in favour of the much simpler include file which just makes it clear 1 db, 1datasource, many tables.

**Comment by ljsinclair** [comment]
Hi https://github.com/andreacappelletti97
thanks for the review.
Fixed the README stuff you pointed out.
FeatureBase overview is (or soon will be) in the site introduction. It's just waiting to be merged.
The trial table has been removed in favour of a simple list of limitations: 1x database, 1x data source, many tables. This makes it clearer and the implication is that if you want more than that, you get a license.
I didn't find the contact us link you mentioned **but** I suspect this was in an earlier version. There's now a working link that points to the help support page

**Comment by ljsinclair** [comment]
cloudfaq.md is fixed in this branch and should be used instead of the one in /main

**Comment by ljsinclair** [comment]
fixed in new-cloud-install PR

**Pull request closed without merging**